src/core/tests/frontend/shared/test_builtin_extensions_1/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "test_builtin_extensions_1")
+
+file(GLOB_RECURSE LIBRARY_SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+file(GLOB_RECURSE LIBRARY_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+
+# Create library
+add_library(${TARGET_NAME} SHARED ${LIBRARY_SRC} ${LIBRARY_HEADERS})
+
+target_link_libraries(${TARGET_NAME} PUBLIC nlohmann_json inference_engine_transformations frontend_common
+        offline_transformations)
+
+add_clang_format_target(${TARGET_NAME}_clang FOR_TARGETS ${TARGET_NAME})
+
```

Comment:
can we test extension w/o creation of so library? E.g. register directly via C++ code.

Reply:
json file contains path to the lib and implemented transformations inside the lib
so I want to check exactly this way: frontend -> JSONConfigExtension -> parse json file -> load so lib
I'm not sure how to emulate this without creation of .so file